GitCommands/Git/GitTagController.cs

```diff
@@ -33,107 +35,171 @@ public static bool CanProvideMessage(this TagOperation operationType)
         }
     }
 
-    public interface IGitTagController
+    public sealed class GitCreateTagArgs
```

Comment:
could you please move it to its own file, it makes the diff hard to comprehend

Reply:
Done.